rascal-lsp/src/main/rascal/util/LanguageServer.rsc

```diff
@@ -49,6 +49,9 @@ alias Builder          = list[Message] (list[loc] /*sources*/, PathConfig /*pcfg
 alias LensDetector     = rel[loc src, Command lens] (Tree /*input*/);
 alias CommandExecutor  = void (Command /*command*/);
 alias InlayHinter      = list[InlayHint] (Tree /*input*/);
+// these single mappers get caller for every request that a user makes, they should be quick as possible
+// carefull use of memo can help with caching
+alias SingleMapper     = set[loc] (loc /*origin*/, Tree /*fullTree*/, Tree /*lexicalAtCursor*/);
```

Comment:
Please change to `Definer` and `Referrer` and `Implementer`, in style with the others (`InlayHinter`). The fact that they all share the same function type is not the point of the alias, and if one of them gets an extra argument in the future, this SingleMapper does not hide anything.

Reply:
okay, I will.